### ISSUE TITLE
No body after upgrading to 0.2 (?)

### ISSUE THREAD

**Issue by nfriedly** [bergos/nodeify-fetch#1]
Hi,

[this code](https://github.com/watson-developer-cloud/speech-javascript-sdk/blob/master/speech-to-text/recognize-file.js#L104) works on nodeify-fetch v0.1.2:

```js
    fetch(options.file)
      .then(function(response) {
        var source = response.body.getReadable();
        source.pipe(recognizeStream);
        streams.unshift(source);
      })
      .catch(function(er) {
        recognizeStream.emit('error', er);
      });
```

But, after I upgraded to v0.2, `response.body` is undefined, and so of course, `response.body.getReadable();` fails - any idea what I'm doing wrong?

**Comment by bergos** [comment]
Sorry, for the breaking and undocumented change. I've updated the example in the README. The API should be now more aligned to the fetch standard. I removed the `get` prefix, because all other methods also don't have it. I also moved it from the `.body` to the response, because `.body` is already the whatwg `ReadableStream`. Also it's now a Promise, because it may require to consume the body, depending on the fallback implementation. I spent some time thinking about it, so I hope it will not change anymore.

**Comment by nfriedly** [comment]
Got it. That makes sense, it does feel a bit more like the native fetch api now.

You might want to do a patch release since https://www.npmjs.com/package/nodeify-fetch still shows the old readme.

**Comment by nfriedly** [comment]
Actually, I think you might want to change the example to


```js
fetch('url')
  .then( response => response.readable() )
  .then( stream => {
    stream.on('data', (chunk) => {
      ...
    })

    stream.on('end', () => {
      ...
    })
  })
```

Saves one layer of indentation + makes error handling a bit easier (one .catch to rule them all).

**Comment by bergos** [comment]
Fixed the example and made a 1.0.1 release. Think it's ready for it and I will use semantic versioning now.

**Comment by nfriedly** [comment]
Cool. Thanks for the help!
